modules/http/scanner.go

```diff
@@ -68,6 +68,10 @@ type Flags struct {
 	// using the specified algorithm, allowing a user of the response to recompute a matching hash
 	ComputeDecodedBodyHashAlgorithm string `long:"compute-decoded-body-hash-algorithm" choice:"sha256" choice:"sha1" description:"Choose algorithm for BodyHash field"`
 
+	// ComputeEncodedBodyHashAlgorithm enables sha256 hashes of the page fingerprint.
+	// Enabled by default. Disabled by default if ComputeDecodedBodyHashAlgorithm is set
+	ComputeEncodedBodyHash bool `long:"compute-encoded-body-hash" description:"Enable sha256 hash for page fingerprint"`
```

Comment:
I suggest naming this `ComputeRawBodyHash`, so it is clear that we are not encoding anything, just computing the hash based on the non-decoded HTTP body.

Reply:
That's more clear, I'll change that over!